src/SFA.DAS.EmploymentCheck.Api.Tests/Application/Controllers/EmploymentCheckControllerTests/WhenPostingRegisterCheck.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmploymentCheck.Api.Application.Controllers;
+using SFA.DAS.EmploymentCheck.Api.Application.Models;
+using SFA.DAS.EmploymentCheck.Api.Mediators.Commands.RegisterCheckCommand;
+
+namespace SFA.DAS.EmploymentCheck.Api.Tests.Application.Controllers.EmploymentCheckControllerTests
+{
+    public class WhenPostingRegisterCheck
+    {
+        private Mock<IMediator> _mediator;
+        private RegisterCheckRequest _registerCheckRequest;
+        private RegisterCheckResult _response;
+
+        [SetUp]
+        public void Setup()
+        {
+            _mediator = new Mock<IMediator>();
+            _registerCheckRequest = new RegisterCheckRequest
+            {
+                CorrelationId = Guid.NewGuid(),
+                CheckType = "CheckType",
+                Uln = 1000001,
+                ApprenticeshipAccountId = 1,
+                ApprenticeshipId = 2,
+                MinDate = DateTime.Today.AddDays(-1),
+                MaxDate = DateTime.Today.AddDays(1),
+            };
```

Comment:
For consistency with the new guidance it would be better to use Autofixture instead of 'new'.